packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -83,142 +88,69 @@ fn get_pkg_path(name: &str, pkg_type: PackageType) -> String {
 
 /// TODO: DOC add docstrings on impl'd methods
 impl<'s> JsPackage<'s> {
-    fn import(
-        scope: &mut v8::HandleScope<'s>,
-        context: v8::Local<'s, v8::Context>,
-        embedded: &Embedded<'s>,
-        name: &str,
-        pkg_type: PackageType,
-    ) -> Result<Self> {
+    fn import(scope: &mut v8::HandleScope<'s>, name: &str, pkg_type: PackageType) -> Result<Self> {
```

Comment:
Rename to `import_package`?

Reply:
Renamed in https://github.com/hashintel/hash/pull/493/commits/16cb022867db4ac17585ff7e9d28a7a9e6862abe